conf/igenomes.config

```diff
@@ -1,421 +1,432 @@
 /*
```

Comment:
I think we can get rid of this file.
rnafusion is not quite working with igenomes anyway

Reply:
6cd2194 should resolve this